package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.9.5",
+  "version": "9.10.0",
```

Comment:
Wondering if this change is big enough for `10.y.z` here ...

Reply:
There is no change to the vmapi functionality (and polling is still used as the fallback, same as before) - so I'm thinking 9.10 is fine.